service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/ReviewCMOServiceTest.java

```diff
@@ -271,4 +337,25 @@ private static HearingBooking hearing(UUID cmoId) {
                 .build())
             .build();
     }
+
+    private CaseData buildCaseData(CMOReviewOutcome cmoReviewOutcome) {
+        return buildCaseData(cmoReviewOutcome, createHearingBookingElements());
+    }
+
+    private CaseData buildCaseData(CMOReviewOutcome cmoReviewOutcome, List<Element<HearingBooking>> hearingDetails) {
+        return CaseData.builder()
+            .state(State.CASE_MANAGEMENT)
+            .reviewCMODecision(ReviewDecision.builder().decision(cmoReviewOutcome).build())
+            .hearingDetails(hearingDetails)
+            .build();
+    }
+
+    private List<Element<HearingBooking>> createHearingBookingElements() {
+        return new ArrayList<>(List.of(
+            element(createHearingBooking(futureDate.plusDays(5), futureDate.plusDays(6), FINAL, cmoID)),
+            element(createHearingBooking(futureDate.plusDays(2), futureDate.plusDays(3), CASE_MANAGEMENT, cmoID)),
+            element(createHearingBooking(futureDate.plusDays(6), futureDate.plusDays(7), ISSUE_RESOLUTION,
+                UUID.randomUUID())),
+            element(createHearingBooking(futureDate, futureDate.plusDays(1), ISSUE_RESOLUTION, cmoID))));
```

Comment:
Similar comment as in differnt place, those are different hearings with the cmoId, I don't fully now it works, but I think it's unexpected, as the initial filtering returns any matching? Thoughts?